go/vt/binlog/keyrange_filter.go

```diff
@@ -5,30 +5,20 @@
 package binlog
 
 import (
-	"encoding/base64"
-	"strconv"
-	"strings"
-
 	log "github.com/golang/glog"
 	"github.com/youtube/vitess/go/vt/binlog/proto"
 	"github.com/youtube/vitess/go/vt/key"
+	annotations "github.com/youtube/vitess/go/vt/sqlannotations"
```

Comment:
It seems like you always import `sqlannotations` as `annotations`. We usually try to avoid renaming on import when possible (except protobufs, where it's conventional), so that the package is always referred to by a single canonical name throughout the codebase. My vote is to call it `sqlannotation` (singular) everywhere.